### DISCUSSION TITLE
DataLink: get the current dashboard path (or URL)

### DISCUSSION THREAD

**Discussion by yarix** [grafana/grafana#64383] [Dashboards, Panels & Visualizations]
when adding links to that points to the current dashboard, it will be helpful to get the current url of the dashboard, or have some other convention to relative urls.
![image](https://user-images.githubusercontent.com/6409857/223677607-17b9c4d5-88a4-4928-90cd-84464419ba73.png)

In the current situation, I need to type the dashboard path/url, and this usually breaks when moving dashboards between Grafana instances or duplication of the dashboard (for development)

### suggestion:
![image](https://user-images.githubusercontent.com/6409857/223675653-73d16c8a-e62f-43d2-bc3b-43514d73d996.png)
it will be helpful the get the current dashboard url dynamically.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82796. Feel free to continue the discussion around this there. Thank you!
